content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,90 @@ output:
 
 ## Introduction
 
-This tutorial will explain how you can match a list of 'scientific names' to the GBIF taxonomic backbone
+This tutorial will explain how you can match a list of _scientific names_ against the [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c).
 
-Important is that you have [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
+Important is that you have the most recent version of [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
 
-```{r message=FALSE, warning=FALSE}
-library(tidyverse) # tidyverse
-library(rgbif)     # To Match GBIF
-library(inborutils) # wrap GBIF api data
+```{r install_pkgs, eval=FALSE}
+devtools::install_github("ropensci/rgbif")    # install rgbif
+devtools::install_github("inbo/inboruilts")   # install inboruilts
+```
+
+```{r load_libraries, message=FALSE, warning=FALSE}
+library(tidyverse)    # tidyverse
+library(rgbif)        # To Match GBIF
+library(inborutils)   # wrap GBIF API data
 library(knitr)
 ```
 
 ## Read data file containing the scientific names
 
 Read file containing the scientific names you want to check against the [GBIF](https://www.gbif.org/species/search?q=) taxonomic backbone:
 
-```{r}
+```{r read_species_list}
 species_list <- read_csv("https://raw.githubusercontent.com/inbo/inbo-pyutils/master/gbif/gbif_name_match/sample.csv", trim_ws = TRUE, col_types = cols())
 ```
    
 Take a look at the data:
 
-```{r}
-knitr::kable(head(species_list))
+```{r preview_species_list}
+knitr::kable(species_list)
 ```
 
-## Request species information
+## Request taxonomic information
 
-Match the column containing the scientificName with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
+Match the column containing the scientific name with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
 
 ```{r, message=FALSE, warning=FALSE}
 species_list_matched <- species_list %>% 
-    gbif_species_name_match(name_col = "name") 
+    gbif_species_name_match(name = "name") 
+```
+
+The `name` argument is the column name containing the scientific names. Default value: `name`. So, the code above is equivalent to:
```

Comment:
Split a paragraph into one line per sentence. While this doesn't affect the end result, it makes diffs more readable. It is easier to spot a change in a sentence than in a paragraph.

Reply:
Thanks. Indeed, your tip helps readibility!
Done here: https://github.com/inbo/tutorials/pull/146/commits/2945283a25e7b7715ad750aeaea163ea0e415053